pkg/chains/storage/grafeas/grafeas.go

```diff
@@ -418,5 +418,10 @@ func (b *Backend) retrieveAllOCIURIs(tr *v1beta1.TaskRun) []string {
 		result = append(result, ref.Name())
 	}
 
+	// for other signable artifacts
+	artifacts := artifacts.ExtractSignableTargetFromResults(tr, b.logger)
+	for _, a := range artifacts {
+		result = append(result, fmt.Sprintf("%s@%s", a.URI, a.Digest))
```

Comment:
I wonder if `StructuredSignable` should have a helper method that returns `fmt.Sprintf("%s@%s", a.URI, a.Digest)`. Otherwise, it will be challenging to keep consistency between the different backends. Wdyt?

Reply:
Good point! Added the new function named `FullRef`.